src/components/CompanyAndJobTitle/TimeAndSalary/EsgBlock/EsgBlock.js

```diff
@@ -24,7 +24,7 @@ const EsgItemBlock = ({
       <Info className={styles.icon} />
     </div>
     <div className={styles.data}>
-      <span className={styles.value}>{value}</span> {unit}
+      <span className={styles.value}>{value.toFixed(1)}</span> {unit}
```

Comment:
Will it be better to be `typeof value === 'number' ? value.toFixed(1) : 'NaN'` 
Just to avoid undefined function error (cause white page)

Reply:
We have defined `value` to be `PropTypes.number.isRequired` so such validation should be redundant.

Reply:
I think react won't really validate props, it will just produce warning message. 
And if it's not a number, it will eventually crash

e.g. try passing undefined, the page will not able to be rendered 
```
         <EsgItemBlock
              className={styles.item}
              title="非主管全時員工薪資平均數"
              year={nonManagerAvgSalaryStatisticsItem.year}
              value={undefined}
              valueCompared={
                nonManagerAvgSalaryStatisticsItem.sameIndustryAverage / 10000
              }
              unit="萬 / 年"
       />
```

Reply:
maybe we can try another solution: https://legacy.reactjs.org/docs/error-boundaries.html

By using `ErrorBoundary` to wrap esg block, we can 

1. prevent the whole page empty (and show corresponding message)
2. log the error

Reply:
NICE this should be in our backlog